README.md

```diff
@@ -139,63 +139,66 @@ mention `std::` or C++. For example, "`<type_traits>`: `is_cute` should be true
 It's okay if you report an apparent STL bug that turns out to be a compiler bug or surprising-yet-Standard behavior.
 Just try to follow these rules, so we can spend more time fixing bugs and implementing features.
 
+# Visual Studio Installer Prerequisites
+
+1. Install [VS 2026 Insiders][] and keep it up to date.
+    + **You must install Insiders for STL development.** The STL and the compiler ship together, and we frequently
+    need the latest compiler fixes and features, so the last production release of the compiler is too old.
+2. Select the "Desktop development with C++" workload.
+3. Select the following components at a minimum:
+    + "MSVC v145 - C++ x64/x86 build tools (Latest)"
+    + "C++ CMake tools for Windows"
+    + "C++ AddressSanitizer"
+    + "Windows 11 SDK (10.0.26100)" or later
+    + "C++ Clang tools for Windows (20.1.8 - x64/x86)"
+    + *Optional:* "MSVC v145 - C++ ARM64/ARM64EC build tools (Latest)"
+4. Install [Python][] 3.13 or later.

```

Comment:
The previous two steps are about installing VS 2026 Insiders, so having these three at the same level is a bit weird.

Reply:
Good point, I'll restructure the lists.